src/core/reactor.ts

```diff
@@ -1279,11 +1287,20 @@ protected _getFirstReactionOrMutation(): Reaction<any> | undefined {
      * @param src
      * @param dst
      */
-    private _disconnect(src: Port<Present>, dst: Port<Present>) {
-        Log.debug(this, () => "disconnecting " + src + " and " + dst);
-        //src.getManager(this.getKey(src)).delReceiver(dst);
-
-
+    protected _disconnect(src: IOPort<Present>, dst: IOPort<Present>) {
+        if (src === undefined || src === null) {
```

Comment:
I don't think it is possible to pass in `undefined` so this check shouldn't be necessary?

Reply:
I redefined the ```src``` and ```dst``` types to match those used in the '_connect' function.